SDK/AppCenterPush/Microsoft.AppCenter.Push.Android/Push.cs

```diff
@@ -53,6 +53,7 @@ static Task PlatformSetEnabledAsync(bool enabled)
         /// <value>
         /// The Sender ID to set.
         /// </value>
+        [Obsolete("For all the Android developers using App Center, there is a change coming where Firebase SDK is required to use Push Notifications. For Android P, its scheduled at the release date for the latest OS version. For all other versions of Android, it will be required after April 2019. Please follow the migration guide at https://aka.ms/acfbxa.")]
```

Comment:
This could be more concise.  I would suggest "There is a change in Android P on release which requires the Firebase SDK to be used for Push notifications.  For other versions, the change will go into effect in April 2019.  Please follow the migration guide at https://aka.ms/acfbxa."

Reply:
As-is, in second sentence, its->it's